apcd-cms/src/apps/submission_form/views.py

```diff
@@ -0,0 +1,25 @@
+from django.http import HttpResponse
+from django.template import loader
+from django.views.generic import View
+from apps.submission_form import apcd_database
+import logging
+import json
+
+logger = logging.getLogger(__name__)
+
+class SubmissionFormView(View):
+    def get(self, request):
+        template = loader.get_template('submission_form/submission_form.test.html')
```

Comment:
**Concern.** I think this should not have `.test`, and the `.test.html` file should be the tiny one, and the `.html` one should be the big.

Reply:
Yep I removed this!